File: Electronics_Shop_17.Model/Requests/AddNews.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddNews
    {
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime DatePublished { get; set; } = DateTime.UtcNow;
    }
}

File: angular17_web/src/app/models/news/add.news.ts
export class AddNews {
  title!: string;
  content!: string;
  datePublished?: Date;

  constructor(data?: any) {
    if (!data) return;

    this.title = data.title;
    this.content = data.content;
    this.datePublished = data.datePublished ? new Date(data.datePublished) : undefined;
  }
}
